From 9655cc2429a94bac062e76678f6a5d421d619455 Mon Sep 17 00:00:00 2001
From: Takaaki Kasai <takaaki.kasai@soliton.co.jp>
Date: Thu, 17 Mar 2016 22:00:21 +0900
Subject: [PATCH 1/2] Change utf-8 symbols of commit graph

---
 src/graph-v2.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/graph-v2.c b/src/graph-v2.c
index 4606105..af73e20 100644
--- a/src/graph-v2.c
+++ b/src/graph-v2.c
@@ -929,11 +929,11 @@ graph_symbol_to_utf8(const struct graph_symbol *symbol)
 {
 	if (symbol->commit) {
 		if (symbol->boundary)
-			return " ◯";
+			return " ◌";
 		else if (symbol->initial)
-			return " ◎";
+			return " ■";
 		else if (symbol->merge)
-			return " ●";
+			return " ◎";
 		return " ●";
 	}
 
-- 
2.9.0

